### PR TITLE
Update dropdowns #265

### LINKED ISSUE #265: Override native UI styling on drop down controls
Where we have in the example screens:

screen shot 2018-02-13 at 1 06 11 pm

vs.

screen shot 2018-02-13 at 1 06 40 pm

We want the control arrows in the first example to be used in the second overriding the browser's native control

additionally the spacing can be much better.

### PR DESCRIPTION
**Pull request by ricardoom**
fixes #265 & #266
@rhernand3z / @ktp242 you're just added on for completion. feel free to review and let me know if you see anything wildly out of order.

### CODE REVIEW

File: docs/discovery-search.html [hunk 1]
```diff
@@ -57,31 +57,7 @@ <h1>Search Results</h1>
...
-                      <option value="date_desc">date (new to old)</option>
-                    </select>
-                    </strong>
-                  </form>
-                  <button id="sort-submission" type="submit" class="">Submit Query</button>
-                </div>
-              </div>
-
-              <div class="nypl-filter-button">
+              <!-- <div class="nypl-filter-button">
```
Comment (EdwinGuzman) [inline docs/discovery-search.html]: Why not just remove all of this?

File: docs/discovery-search.html [hunk 2]
```diff
@@ -112,13 +88,41 @@ <h3>
...
+          <div class="nypl-column-full">
+          <div class="nypl-results-summary" aria-live="assertive" aria-atomic="true">
+            <h2>Displaying 1-50 of 843 results for keyword "New York Waterways"</h2>
+          </div>
+          <div class="nypl-results-sorting-controls">
+            <div class="nypl-results-sorter">
+              <div class="nypl-select-field-results">
+                <label for="sort-by-label">Sort by</label>
+                <form action="#" method="POST">
+                  <span class="nypl-omni-fields">
```
Comment (EdwinGuzman) [inline docs/discovery-search.html]: You're missing a closing `span`.

File: docs/discovery-search.html [hunk 3]
```diff
@@ -112,13 +88,41 @@ <h3>
...
+                      <option value="date_asc">date (old to new)</option>
+                      <option value="date_desc">date (new to old)</option>
+                    </select>
+                  </strong>
+                </form>
+              <button id="sort-submission" type="submit" class="">Submit Query</button>
+            </div> <!-- nypl-select-field-results -->
+          </div><!-- /nypl-results-sorter -->
+        </div><!-- /nypl-results-sorting-controls -->
+      </div>
```
Comment (EdwinGuzman) [inline docs/discovery-search.html]: `<!-- /nypl-column-full -->`

Comment (EdwinGuzman) [reply docs/discovery-search.html]: This can also be indented, which means some of the DOM in the `<select>` are not indented properly.

**Comment by EdwinGuzman** [comment]
For the sorter, the description and select dropdown are not vertically aligned to each other.
screen shot 2018-02-14 at 4 19 06 pm

File: docs/discovery-search.html (hunk 1 above)
Comment (ricardoom) [reply docs/discovery-search.html]: gone

File: docs/discovery-search.html (hunk 2 above)
Comment (ricardoom) [reply docs/discovery-search.html]: fixed

File: docs/css/example-search-styles.css [hunk 4]
```diff
@@ -3722,7 +3727,7 @@ dl dd.multi-item-list .nypl-basic-table,
 
 .nypl-results-summary {
   display: inline-block;
-  max-width: 65%; }
+  flex-grow: 2; }
```
Comment (EdwinGuzman) [inline docs/css/example-search-styles.css]: Fancy 🎩

File: sass/_tables-results.scss [hunk 5]
```diff
@@ -47,29 +47,26 @@ $results-item-border-width: 0.0625rem;
 
 @mixin results-summary {
   @include clearfix;
+  @include media($mobile-breakpoint) {
```
Comment (EdwinGuzman) [inline sass/_tables-results.scss]: Not needed?

File: sass/_tables-results.scss [hunk 6]
```diff
@@ -47,29 +47,26 @@ $results-item-border-width: 0.0625rem;
...
-
-  @include media($mobile-breakpoint) {
-    //max-width: 49%;
-    //float: left;
-  }
-
   display: inline-block;
-  //float: left;
-  max-width: 65%;
+  flex-grow: 2;
```
Comment (EdwinGuzman) [inline sass/_tables-results.scss]: 🎩

File: docs/css/example-search-styles.css (hunk 4 above)
Comment (ricardoom) [reply docs/css/example-search-styles.css]: maybe I should use `@supports` and keep it in?

**Comment by EdwinGuzman** [comment]
Ah, and fix tests.

File: docs/css/example-search-styles.css (hunk 4 above)
Comment (EdwinGuzman) [reply docs/css/example-search-styles.css]: It'd be good to test the PR in multiple browsers and then see if we do need to use `@supports`. But, I'm liking it and don't think it'd hurt to check for it. You'd then need another set of rules for when it isn't supported, no?

Comment (ricardoom) [reply docs/css/example-search-styles.css]: yeah. we'd basically have to have the previous work as the fall back.

File: sass/_tables-results.scss (hunk 5 above)
Comment (ricardoom) [reply sass/_tables-results.scss]: nope. removed.

**Pull request merged**